thirdeye/thirdeye-frontend/app/pods/components/alert-details/template.hbs

```diff
@@ -40,102 +29,88 @@
+    {{#unless isPreviewLoading}}
+      {{#unless disablePreviewButton}}
+      <div class="col-xs-12 preview">
+        <div class="col-xs-2 no-padding">
+          <label class="te-label te-label--display">Display Window
+            <span>
+              <i class="glyphicon glyphicon-question-sign"></i>
+              {{#tooltip-on-element class="te-tooltip"}}
+                This is the maximum display period in days. You can zoom into the chart further using the mouse wheel.
+              {{/tooltip-on-element}}
+            </span>
+          </label>
+        </div>
+        <div class="col-xs-4 no-padding">
+          {{date-range-picker
+            class="te-range-picker"
+            alwaysShowCalendars=true
+            timePicker=true
+            timePicker24Hour=true
+            timePickerIncrement=pill.timePickerIncrement
+            start=pill.activeRangeStart
+            end=pill.activeRangeEnd
+            ranges=pill.predefinedRanges
+            showCustomRangeLabel=false
+            format=pill.uiDateFormat
+            serverFormat="YYYY-MM-DD HH:mm"
+            applyAction=(action "onRangeSelection")
+          }}
+        </div>
+      </div>
+      {{/unless}}
+    {{/unless}}
+    {{!-- draw chart & table--}}
+    {{#if showDetails}}
+      {{#if dataIsCurrent}}
+        {{#unless isPreviewLoading}}
+            {{!-- Stats Cards --}}
+            <div class="te-content-block__performance-health-wrapper">    
+              <!-- Anomalies, Response Rate, Precision, Recall -->
+              <div class="te-horizontal-cards te-content-block{{if alertId '__alert-performance'}}" >
+                <h4 class="te-self-serve__block-title">
+                  <label for="select-dimension" class="control-label te-label">
+                    Alert Performance
+                    <span>
+                      <i class="glyphicon glyphicon-question-sign"></i>
+                      {{#tooltip-on-element class="te-tooltip"}}
+                        All estimated performance numbers are based on reviewed anomalies.
+                      {{/tooltip-on-element}}
+                    </span>
+                  </label>
+                </h4>
+                {{!-- Alert anomaly stats cards --}}
+                {{#if performanceStatsTaskInstance.isRunning }}
+                  <div class="spinner-wrapper spinner-wrapper--card">
+                    {{ember-spinner lines=30 radius=20 length=0 width=10 opacity=0 trail=75 color='blue'}}
+                  </div>
+                {{else}}  
+                  {{stats-cards
+                    stats=computedStats
+                    areTwoSetsOfAnomalies=areTwoSetsOfAnomalies
+                    isEditMode=isEditMode
+                  }}
+                {{/if}}
+              </div>
+              {{#if alertId}}
+                {{detection-health
+                health=detectionHealth
+                selectedRule=selectedRule
                 }}
               {{/if}}
```

Comment:
Too many nested if statements, we could consider reorganizing the logic.

Reply:
Agreed.  That said, this is an improvement over the original template. 
I'll take a look at reducing the nested ifs but I can't promise much improvement at this stage
- the functional refactoring may offer an opportunity to move the conditionals from the `template` 
-into the `component/controller`